source/general/manifest/manifest.1.json

```diff
@@ -127,6 +133,12 @@
       "license": { "type": "string", "description": "The license of the pack.", "title": "License" },
       "url": { "type": "string", "description": "The home website of your pack.", "title": "Url" },
       "description": "UNDOCUMENTED: metadata.",
+      "product_type": {
+        "type": "string",
+        "title": "Product Type",
+        "description": "The type of product this pack is. This is used to determine how the pack is displayed in the store.",
+        "enum": ["", "addon"]
+      },

```

Comment:
You sure this need to be on version 1 of the manifest and not two 🤔

Reply:
It should work for either version, otherwise please rectify if I'm wrong.

But besides that, when I was inspecting the schemas, these properties weren't in there yet — only to realize that my fork was outdated upon merging.